.github/workflows/checks.yml

```diff
@@ -92,6 +92,8 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Rust Cache
         uses: Swatinem/rust-cache@v2
+      - name: Check TOML
+        uses: dprint/check@v2.2
```

Comment:
I am fine with dprint fmt for now, but a new check can be annoying.

Reply:
I have it installed & my editor would automatically execute it after i save files. You will not feel it. Or i think we can set it up pre-commit. wdyt?

Reply:
lets go the precommit way

Reply:
Added